benchmarks/transformer/score_mod.py

```diff
@@ -98,7 +99,7 @@ def generate_inputs(
     return query, key, value
 
 
-def run_single_experiment(config: ExperimentConfig) -> ExperimentResults:
```

Comment:
Above in this file is 
```
torch._dynamo.config.automatic_dynamic_shapes = False
```
does compile ignore this if dynamic=true?

Reply:
Yes, ```dynamic=True``` means forcing dynamic.